src/actions/types.js

```diff
@@ -4,6 +4,6 @@ export const BEERS_FETCH_FAILURE = 'BEERS_FETCH_FAILURE';
 export const UPDATE_SEARCH_RESULT = 'UPDATE_SEARCH_RESULT';
 export const SET_BEER_ID = 'SET_BEER_ID';
 export const UPDATE_HOME_BEERS = 'UPDATE_HOME_BEERS';
-export const UPDATE_FAVOURITES = 'UPDATE_FAVOURITES';
+export const UPDATE_FAVOURITES = 'UPDATE_FAVOURITES'; // Too generic
```

Comment:
Just curious, Why do you think is generic?

Reply:
If you have `UPDATE_FAVOURITES` for the beer reducers and you also have `UPDATE_FAVOURITES` for say the clubs reducers, When you dispatch the action, both reducers will be called
That is why some devs do stuffs like
```suggestion
export const UPDATE_FAVOURITES = '@BEERS_/UPDATE_FAVOURITES'; // Too generic
```
Just any weird syntax to prefix the actions so that they don't get marshed up